app/build.gradle

```diff
@@ -192,6 +192,24 @@ dependencies {
 
     // TicketSale
     implementation 'com.stripe:stripe-android:6.1.2'
+
+
+    // Dependencies for Android instrumented unit tests
+    androidTestImplementation "junit:junit:4.13.2"
+    // Testing code should not be included in the main code.
+    // Once https://issuetracker.google.com/128612536 is fixed this can be fixed.
+
+    debugImplementation "androidx.fragment:fragment-testing:1.3.6"
```

Comment:
This dependency breaks my debug build: When I switch the activity (e.g. open the calendar), the top navigation buttons disappear, are you experiencing this as well?
Does `testImplementation` work for you as well?

Reply:
Yes, I was also exeriencing this problem and both testImplementation and androidTestImplementation fixed it. [However **according to Google** one is supposed to use **debugImplementation.**](https://developer.android.com/guide/fragments/test#declare-dependencies)

Since testing is a bigger issue anyways (as discussed in Discord), I would **rollback app/build.gradle** and merge my existing PRs as is, without further testing/dependencies and take care of that in another PR. Would that be okay with you?

Reply:
That sounds like the best approach.